workers/redirector.js

```diff
@@ -26,8 +32,8 @@ const experimentPages = [];
  */
 
 const workerPaths = {
-    'staging': [],
-    'prod': []
+    'staging': ['https://www.allizom.org/en-US/*', 'https://www.allizom.org/de/*'],
```

Comment:
do you want/need german pages to go to this worker?

Reply:
I don't need, I was reverting my previous change. Happy to remove.